specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/examples/expressRouteProviderPortList.json

```diff
@@ -0,0 +1,49 @@
+{
+    "parameters": {
+      "api-version": "2021-08-01",
+      "subscriptionId": "subid"
+    },
+    "responses": {
+        "200": {
+          "body": {
+            "value": [
+              {
+                "portPairDescriptor": "bvtazureixpportpair1",
```

Comment:
>"portPairDescriptor": "bvtazureixpportpair1", [](http://example.com/codeflow?start=16&length=45)

Invalid, according to your definition this is part of your "properties" bag. There's a well known set of top level properties (e.g. id, type), cannot add another arbitrary top-level property.

Reply:
fix in your examples please.

Reply:
done